packages/grafana-ui/src/components/uPlot/config/UPlotScaleBuilder.ts

```diff
@@ -104,6 +111,9 @@ export class UPlotScaleBuilder extends PlotConfigBuilder<ScaleProps, Scale> {
         minMax = uPlot.rangeNum(hardMinOnly ? hardMin : dataMin, hardMaxOnly ? hardMax : dataMax, rangeConfig);
       } else if (scale.distr === 3) {
         minMax = uPlot.rangeLog(dataMin!, dataMax!, scale.log ?? 10, true);
+      } else if (scale.distr === 4) {
+        // TODO: fix fullMags: true (https://github.com/leeoniya/uPlot/issues/749)
```

Comment:
this one's on me ;)

Reply:
fixed in https://github.com/leeoniya/uPlot/commit/de01c7465f50573971976fea2e94b11e96fd2760